### PR TITLE
[EDMTP-396] Added ability to display empty folders in console

### PR DESCRIPTION
**Pull request by yminkevich**

### CODE REVIEW

File: core/src/main/java/com/exadel/aem/toolkit/core/lists/servlets/ListsServlet.java [hunk 1]
```diff
@@ -132,15 +133,28 @@ private static boolean isList(ResourceResolver resolver, Resource resource) {
...
 
     /**
-     * Checks whether the resource contains resources, which are either pages, or folders that may contain lists inside
+     * Checks whether the resource is a folder
      *
      * @param resource {@code Resource} instance used as the source of markup
      * @return True or false
      */
     private static boolean isFolder(Resource resource) {
+        String primaryType = resource.getValueMap().get(JcrConstants.JCR_PRIMARYTYPE, String.class);
```
Comment (liubou-masiuk) [inline core/src/main/java/com/exadel/aem/toolkit/core/lists/servlets/ListsServlet.java:142]: (cosmetic) If you provide a default while retrieving values ( `resource.getValueMap().get(JcrConstants.JCR_PRIMARYTYPE, StringUtils.EMPTY);` , you can omit null-check
also, [yoda conditions](https://en.wikipedia.org/wiki/Yoda_conditions)  to make your code npe-safe

**Review by liubou-masiuk — APPROVED** [review]
👍

File: core/src/main/java/com/exadel/aem/toolkit/core/lists/servlets/ListsServlet.java (hunk 1 above)
Comment (smiakchilo) [reply core/src/main/java/com/exadel/aem/toolkit/core/lists/servlets/ListsServlet.java:142]: That's right!
Or even:
`StringUtils.equalsAny(myString, "firstValriant", "secondVariant", "thirdValriant")`
(null check out of the box)

**Review by smiakchilo — APPROVED** [review]

File: core/src/main/java/com/exadel/aem/toolkit/core/lists/servlets/ListsServlet.java (hunk 1 above)
Comment (yminkevich) [reply core/src/main/java/com/exadel/aem/toolkit/core/lists/servlets/ListsServlet.java]: Thank you for good point

**Pull request merged**